### PR TITLE
Added unit test

### PR DESCRIPTION
**Pull request by Atalay-Polat**
Unit test for UserService method

### CODE REVIEW

File: exalearn-api/ExaLearn.Bl/Mapping/UserProfile.cs [hunk 1]
```diff
@@ -27,6 +27,15 @@ public UserProfile()
                 .ForMember(x => x.FullName, map => map.MapFrom(source => $"{source.User.FirstName} {source.User.LastName}"))
                 .ReverseMap()
                 .ForAllOtherMembers(x => x.Ignore());
+
+            CreateMap<AssignTest, HRAssignedTestDTO>()
+                .ForMember(x => x.Level, map => map.MapFrom(source => source.LevelType))
+                .ForMember(x => x.ExpireDate, map => map.MapFrom(source => source.ExpirationDate))
+                .ForMember(x => x.AssignedBy, map => map.MapFrom(source => $"{source.Assigner.FirstName} {source.Assigner.LastName}"))
+                .ForMember(x => x.Passed, map => map.Ignore())
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Bl/Mapping/UserProfile.cs:35]: why ignore

File: exalearn-api/ExaLearn.Tests/Services/UserServiceTests.cs [hunk 2]
```diff
@@ -120,5 +152,35 @@ public async Task GetHrUserHistoryByIdAsync_HrHistroryModelIsValid_ResultWithVal
             Assert.Equal("Aaron Ramsdale", result[0].FullName);
             Assert.Equal("Sam Johnstone", result[1].FullName);
         }
+
+        [Fact]
+        public async Task GetHrAssignedTestByIdAsync_HrAssignTestModelIsValid_ResultWithValidLevel()
+        {
+            // Arrange
+            var tests = _mockAssignTestRepository.Object.GetByIdAsync(1);
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Tests/Services/UserServiceTests.cs]: why tests if it is one assigntest

File: exalearn-api/ExaLearn.Bl/Mapping/UserProfile.cs (hunk 1 above)
Comment (Atalay-Polat) [reply exalearn-api/ExaLearn.Bl/Mapping/UserProfile.cs:35]: > why ignore

There is no field Passed in AssignTest

**Review by romanstat — APPROVED** [review]

File: exalearn-api/ExaLearn.Tests/Services/UserServiceTests.cs [hunk 3]
```diff
@@ -120,5 +152,35 @@ public async Task GetHrUserHistoryByIdAsync_HrHistroryModelIsValid_ResultWithVal
...
+            var test = _mockAssignTestRepository.Object.GetByIdAsync(1);
+
+            // Act
+            var result = await _userService.GetHrAssignedTestByIdAsync(test.Id);
+
+            // Assert
+            Assert.NotNull(result);
+            Assert.Equal("Bob Holland", result[0].AssignedBy);
+            Assert.Equal("Joe Howard", result[1].AssignedBy);
+        }
```
Comment (artemgael96) [inline exalearn-api/ExaLearn.Tests/Services/UserServiceTests.cs:171]: make two tests to one test
Make Assert region: 
Assert.NotNull(result);
Assert.Equal(LevelType.Beginner, result[0].Level);
Assert.Equal("Bob Holland", result[0].AssignedBy);
Assert.Equal(LevelType.Intermediate, result[1].Level);
Assert.Equal("Joe Howard", result[1].AssignedBy);

Comment (Atalay-Polat) [reply exalearn-api/ExaLearn.Tests/Services/UserServiceTests.cs:171]: Done

**Review by ExadelSandbox — APPROVED** [review]

**Pull request merged**